src/k8s/go.mod

```
@@ -1,31 +1,31 @@
 module github.com/canonical/k8s
 
-go 1.22.6
+go 1.22.7
 
 require (
 	dario.cat/mergo v1.0.0
-	github.com/canonical/go-dqlite v1.22.0
+	github.com/canonical/go-dqlite/v3 v3.0.0
```

```suggestion
	github.com/canonical/go-dqlite/v2 v2.0.0
```

We should use `v2` here for our direct dependency.